ext/liquid_c/vm.c

```diff
@@ -67,8 +68,6 @@ void init_liquid_vm()
     id_render_node = rb_intern("render_node");
     id_ivar_interrupts = rb_intern("@interrupts");
     id_ivar_resource_limits = rb_intern("@resource_limits");
-    id_ivar_render_length_limit = rb_intern("@render_length_limit");
-    id_ivar_last_capture_length = rb_intern("@last_capture_length");
     id_increment_render_score = rb_intern("increment_render_score");
     id_increment_write_score = rb_intern("increment_write_score");
     id_raise_limits_reached = rb_intern("raise_limits_reached");
```

Comment:
These shouldn't be needed here either, since we can instead do C function calls now that we have a resource limits struct.